docs/raw/organisation-channels.md

```diff
@@ -0,0 +1,160 @@
+---
+permalink: /spec/17
+parent: Stable specs
+title: 17/ORGANISATION-CHANNELS
+---
+
+# 17/ORGANISATION-CHANNELS
+
+> Version: 0.1
+>
+> Status: Raw
+>
+> Authors: Andrea Maria Piana <andreap@status.im>
+>
+
+- [Organisation channels](#17organisation-channels)
+  - [Reason](#reason)
+  - [Fetching information about an organisation](#fetching-information-about-an-organisation)
+  - [Joining an organisation](#joining-an-organisation)
+  - [Updating channels](#updating-channels)
+
+## Reason
+
+Organisation channels are a way to allow users to build communities and create curated content.
+
+## Fetching information about an organisation
+
+An organisation is uniquely identified by a public key.
+
+This public key can be discovered through different means, links, indexable web pages or search directories for example.
+
+Given that the public key of the organisation is known, information about the channels and members can be fetched by listening to a discovery topic derived by the public key:
+
+```
+   Does not matter which topic for now, as long as it does not clash with public chats.
```

Comment:
How can this be assured? Or is this more of a soft guarantee ("don't use a topic that corresponds to ASCII string (e..g)"?

What happens if someone else decided to use that topic in the network? (I assume nothing but some extra BW, just to make sure)

Reply:
So a filter would be a combination of topic + encryption key.
For this channels we should not re-use the same algorithm for calculating those, mainly for bw issues.
If there's a topic only clash (different encryption keys), that's not an issue as we would have two different filters handling this.
If there's both a topic and encryption key clash (that's completely avoidable on our side), then we might start having issues, but `chat-id` is in the message so I believe we can still differentiate them.